pyproject.toml

```
@@ -123,23 +224,21 @@ profile = "black"
 force_grid_wrap = 1
 
 [tool.ruff]
-target-version = "py37"
 select = [
     "E", # errors
     "F", # pyflakes
     "D", # pydocstyle
     "UP", # pyupgrade
+    "C4", # flake8-comprehensions
+    "RUF", # ruff
+    "NPY", # numpy
```

I disable it until you can fix it
```suggestion
    # "NPY", # numpy
```